nemo/collections/asr/data/audio_to_text.py

```diff
@@ -132,6 +132,7 @@ def __init__(
         bos_id: Optional[int] = None,
         eos_id: Optional[int] = None,
         pad_id: int = 0,
+        manifest_line_parser: Callable[[str, Optional[str]], Dict[str, Any]] = None,
```

Comment:
This should be Optional[Callable...]

Reply:
It should be the same as here:
https://github.com/NVIDIA/NeMo/blob/9c75c348bc8137717ca79a34a93c50028ffdfd38/nemo/collections/asr/parts/manifest.py#L35

Reply:
Hmm the signature and how we use it is different. For now I guess its fine to leave as it is, it becomes way too long to add on Union[Optional[...], str].